apps/application-system/api/src/app/modules/application/files/file.service.spec.ts

```diff
@@ -1,114 +1,191 @@
 import { Test } from '@nestjs/testing'
 import { FileService } from './file.service'
-import { FormValue } from '@island.is/application/core'
-import {
-  ParentResidenceChange,
-  PersonResidenceChange,
-} from '@island.is/application/templates/children-residence-change'
+import { SigningService, SIGNING_OPTIONS } from '@island.is/dokobit-signing'
+import * as aws from './utils/aws'
+import * as pdf from './utils/pdf'
+import { Application } from './../application.model'
+import { ApplicationTypes, PdfTypes } from '@island.is/application/core'
+import { LoggingModule } from '@island.is/logging'
+import { NotFoundException } from '@nestjs/common'
 
 describe('FileService', () => {
   let service: FileService
+  let signingService: SigningService
+  const signingServiceRequestSignatureResponse = {
+    controlCode: 'code',
+    documentToken: 'token',
+  }
+  const applicantsPhoneNumber = '111-2222'
```

Comment:
I am wondering if we want to have all the test data in the same place. I get that the data that is used in more than place has been extracted to a constant but in my opinion it would be super nice to have all the data in one place and constructed with an object:

```
const parentA: {
    phoneNumber: '111-2222',
    name: 'Test name',
}
const parentB: {
    phoneNumber: '222-1111',
    name: 'Parent B name',
}
```
etc.